### PR TITLE
top erc20s

### PR DESCRIPTION
**Pull request by andreipope**
top erc20s

### CODE REVIEW

File: docs/.vuepress/public/json/topTokensList.json [hunk 1]
```diff
@@ -0,0 +1,1299 @@
+{
+  "tokens": [
+    {
+      "filename": "PAXToken",
+      "name": "Paxos Standard",
+      "decimal": 18,
+      "pricePerCent": 1,
+      "symbol": "PAX",
+      "address": {
+        "1": "0x8e870d67f660d95d5be530380d0ec0bd388289e1",
```
Comment (mattkanwisher) [inline docs/.vuepress/public/json/topTokensList.json]: not sure it makes sense to put all these other networks in the json

File: docs/en/top-erc20s.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+id: top-erc20
+title: Top ERC20s Deployed to Loom
+sidebar_label: Top ERC20s Deployed to Loom
+---
+
+Here's the list of the top 100 ERC20 smart contracts already deployed to Loom. Please scroll to the right to see all columns.
+
+| Token Name  | Decimals | MainNet | asia1 | stage | rinkeby-asia1 | rinkeby-us1 | plasma |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+Paxos Standard | 18 | 0x8e870d67f660d95d5be530380d0ec0bd388289e1 | 0x65250dc7016addd1ce226259e2e60ca8ac13ca52 | 0x56c80de2eafe0086d690b87082472548f7bb917f | 0x9d633be2ff4151e85fc5d6444b5e12f5ae442f23 | 0x08eff0e92291b70cdb5c1cfe81352aca669a9e19 | 0xd443c9cee967b5d068dbf7471c33a2e0bd135f97 |
```
Comment (mattkanwisher) [inline docs/en/top-erc20s.md]: ONLY plasma chain, please remove all other networks in the markdown

Comment (andreipope) [reply docs/en/top-erc20s.md]: > ONLY plasma chain, please remove all other networks in the markdown

@mattkanwisher the list is updated :)

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -188,6 +188,14 @@ module.exports = {
           ['/non-validator-node', 'Non Validator Node'],
         ],
       },
+      {
+        title: 'Top ERC20s',
```
Comment (mattkanwisher) [inline docs/.vuepress/config.js]: wouldn't create any more top level sections, lets move this to the other section and maybe link to it in FAQ

Comment (andreipope) [reply docs/.vuepress/config.js]: > wouldn't create any more top level sections, lets move this to the other section and maybe link to it in FAQ

@mattkanwisher I just moved it to FAQ :). Couldn't find a better place.

**Pull request merged**